pkg/operator/ceph/object/bucket/util.go

```diff
@@ -47,12 +48,13 @@ const (
 	objectStoreName      = "objectStoreName"
 	objectStoreNamespace = "objectStoreNamespace"
 	objectStoreEndpoint  = "endpoint"
+	threadcount          = "5"
 )
 
 func NewBucketController(cfg *rest.Config, p *Provisioner) (*provisioner.Provisioner, error) {
 	const allNamespaces = ""
 	provName := cephObject.GetObjectBucketProvisioner(p.context, p.namespace)
-
+	os.Setenv("LIB_BUCKET_PROVISIONER_THREADS", threadcount)
```

Comment:
I would think this would be set in the env (via the manifest) rather than hard coded.

Reply:
these are actually goroutines rather than threads, right? I assume we're just tied to the name of the env var from the bucket provisioner.

Reply:
yes to goroutines but they are almost threads. I guess some are calling them _green_ threads: http://tleyden.github.io/blog/2014/10/30/goroutines-vs-threads/
The env var could have been named `LIB_BUCKET_CONCURRENCY_COUNT` to be more accurate.

Reply:
I agree with @copejon  that this env var should be set in the rook operator manifest, not in the code here.